tests/datasets/test_etci2021.py

```diff
@@ -72,7 +72,10 @@ def test_getitem(self, dataset: ETCI2021) -> None:
             assert x["mask"].shape[0] == 1
 
     def test_len(self, dataset: ETCI2021) -> None:
-        assert len(dataset) == 2
+        if dataset.split == "train":
+            assert len(dataset) == 12
+        else:
+            assert len(dataset) == 4
```

Comment:
Would it be possible to reduce the number of test files (dataset length)? There are quite a lot.